README.md

```diff
@@ -1,58 +1,68 @@
 
-## License
+## Documentation
+
+We encourage developers to read the documentation before contributing. Understanding the definitions and the architecture of the project will help you understand the codebase and make it easier for you to contribute to the project.
 
+For more details about this project, [view the documentation](https://edehr.github.io/edehr/)
+
+Alternatively, you can run a local documentation server by running the following command in your terminal:
+```
+npm run docs:dev
+```
+Then visit  [http://localhost:8080/edehr](http://localhost:8080/edehr) See documentation in the ```docs``` directory.
 
-The master-bccampus branch is released under a copyright licence (MIT). This includes all work funded by BC Campus from project start, September 2018 to project end, January 31, 2020.
 
-Work since the BC Campus project is released under a copyleft license (GNU Affero General Public License).
 
+## Development setup
 
-## Blog Posts
+The following section will walk you through the process of installing and running the project in your local machine. It will also guide you on how to run the tests and how to run the project in a docker container.
 
-[BC Campus announcement](https://bccampus.ca/2019/02/07/an-open-educational-tool-for-training-students-to-use-electronic-health-record-systems-in-the-workplace/) By E Beattie, February 7, 2019
 
+## Pre-requisites
 
-[BC Campus Blog - An Education in Electronic Health Record](https://bccampus.ca/2020/01/07/an-education-in-electronic-health-records/) By J Caldwell, January 7, 2020
- 
-## Documentation
+* In order to be able to run this project, you will need to do some simple set-up. This section will guide you on everything that will be expected in order to run it.
 
-Public docs are available here [https://edehr.github.io/edehr](https://edehr.github.io/edehr)
+* The first step is making sure you have a working version of git installed on your machine. You can find the installation instructions [here](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git).
 
-## Demo
+* The second step is to make sure you have a working version of npm and Node.js installed on your machine. You can find the installation instructions [here](https://nodejs.org/en/download/).
 
-A demonstration mode is currently being implemented. 
-This demo mode mimics a learning management system allowing you to experience both the student's  and the instructor's experiences. 
-For more information write an email to [info@edehr.org](mailto:info@edehr.org). 
+* (Optional) The third step is to make sure you have a working version of Docker installed on your machine. You can find the installation instructions [here](https://docs.docker.com/engine/install/).
```

Comment:
It is optional only if the reader can set up EdEHR to access a local instance of MongoDB.  Otherwise this project uses Docker to run an instance of MongoDB.  We use a container so that this instance of mongo is kept separate from any other instances.